lib/core/seleniumRunner.js

```diff
@@ -389,9 +389,15 @@ class SeleniumRunner {
    * @param {*} name - the name of the script
    */
   async runScriptFromCategory(script, isAsync, name) {
+    const privileged = (name && name.startsWith('privileged'));
+
     // Scripts should be valid statements or IIFEs '(function() {...})()' that can run
     // on their own in the browser console. Prepend with 'return' to return result of statement to Browsertime.
     if (isAsync) {
+      if (privileged) {
+        throw new BrowserError(`Asynchronous privileged JavaScript is not yet supported: cannot run script ${name}`);
```

Comment:
This is now supported.

Reply:
Good point.  I'm going to push this out to a separate landing; the commit after it is "don't do it, 'cuz it breaks Chrome".  We'll work with Benjamin to get this in as follow-up.